src/SFML/Graphics/Sprite.cpp

```diff
@@ -61,20 +61,19 @@ Sprite::Sprite(const Texture& texture, const IntRect& rectangle) : m_texture(nul
 void Sprite::setTexture(const Texture& texture, bool resetRect)
 {
     // Recompute the texture area if requested, or if there was no valid texture & rect before
-    if (resetRect || (!m_texture && (m_textureRect == sf::IntRect())))
+    if (resetRect || (!m_texture && (m_textureRect == IntRect())))
     {
         setTextureRect(IntRect({0, 0}, Vector2i(texture.getSize())));
     }
 
-    // Assign the new texture
     m_texture = &texture;
 }
 
 
 ////////////////////////////////////////////////////////////
 void Sprite::setTextureRect(const IntRect& rectangle)
 {
-    if (rectangle != m_textureRect)
+    if (m_textureRect != rectangle)
```

Comment:
Why did you make this? Seems like more unnecessary refactoring. These changes clutter up the PR and slow down the review.